docs/content/how-to/agent/create-heterogeneous-nodepools.md

```diff
@@ -0,0 +1,166 @@
+# Create Heterogeneous NodePools on Agent HostedClusters
+
+This document explains how to create heterogeneous nodepools on agent platform. 
+Please [refer](create-agent-cluster.md) to set up the env for agent cluster, this document only covers the things you need to configure to have heterogeneous node pools.
+
+## Configure AgentServiceConfig with two heterogeneous architecture OS images
+
+~~~sh
+export DB_VOLUME_SIZE="10Gi"
+export FS_VOLUME_SIZE="10Gi"
+export OCP_VERSION="4.15.0"
```

Comment:
Should this not be 4.16?

Reply:
4.16 stable ISOs not released yet, wanted to document the stable one, hence mentioned 4.15.